trade-finance-manager-ui/server/controllers/case/amendments/amendmentFacilityEndDate.get.api-test.js

```diff
@@ -199,6 +199,108 @@ describe('amendmentFacilityEndDate routes', () => {
           user,
         });
       });
+
+      it("should render the template with the current facility date as undefined if facility end date is not in the facility snapshot and there hasn't yet been an amendment to it", async () => {
+        api.getAmendmentById.mockResolvedValueOnce({
+          status: 200,
+          data: {
+            ...MOCK_AMENDMENT_COVERENDDATE_CHANGE_USING_FACILITY_ENDDATE,
+          },
+        });
+        api.getLatestCompletedAmendmentFacilityEndDate = jest.fn().mockResolvedValueOnce({ status: 200, data: {} });
+        api.getFacility = jest.fn().mockResolvedValueOnce({ facilitySnapshot: { dates: {} } });
+
+        const req = {
```

Comment:
Not worth changing here, but just a note that you can use `node-mocks-http` to create mock request and response objects:
```typescript
import httpMocks from 'node-mocks-http';

const { req, res } = httpMocks.createMocks({
  session,
  params: {
    _id: dealId,
    amendmentId,
    facilityId,
  },
});
```

Reply:
Oh nice, thank you! I won't change here but will bear in mind in future! 👍